Overview.src.html

```diff
@@ -6584,7 +6586,24 @@ <h4 id="processing-a-keyframe-object">Processing a <code>Keyframe</code> object<
 
 </div>
 
+<h3 id="the-sharedkeyframelist-interface">The <code>SharedKeyframeList</code> interface</h3>
+
+The {{SharedKeyframeList}} interface allows lists of {{Keyframe}} dictionaries
+to be represented.
+
+Note: This means that multiple {{KeyframeEffect}} objects can re-use the same
+{{Keyframe}}s without paying the cost of parsing them multiple times.
+
+If constructed from a {{Keyframe}} or a sequence, the provided parameter is
+cloned and stored internally. If constructed from another 
+{{SharedKeyframeList}}, then a reference to the internal {{Keyframe}} or
+sequence is stored.
 
+<pre class='idl'>
+[Constructor (Keyframe or sequence&lt;Keyframe&gt; or SharedKeyframeList) frames)]
+interface SharedKeyframeList {
+};
+</pre>
```

Comment:
Done, I think.

I currently have new SharedKeyframeList(otherSharedKeyframeList) sharing the reference, but cloning is OK too.